docs/backups.md

```diff
@@ -363,7 +445,23 @@ restoration can be done in the following way.
                 credentialsSecret: my-cluster-name-backup-s3
                 region: us-west-2
                 endpointUrl: https://URL-OF-THE-S3-COMPATIBLE-STORAGE
+                ...
+            ```
+
+        === "Azure Blob storage"
+
+            The `backupSource` key should contain `destination` key equal to the
+            Azure Blob [container](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#containers)
+            with `https://` prefix, followed by the necessary Azure
+            configuration keys, same as in `deploy/cr.yaml` file:
+
+            ```yaml
             ...
+            backupSource:
+              destination: https://AZURE-CONTAINER-NAME/BACKUP-NAME
```

Comment:
@nmarukovich please verify this

Reply:
The whole way looks like this 
https://engk8soperators.blob.core.windows.net/operator-testing/cluster1-2022-11-21-17:51:07-full

operator-testing = AZURE-CONTAINER-NAME

Reply:
This should not work with `https://` prefix

Reply:
@pooknull is `https://` not necessary or just wrong?

Reply:
```
apiVersion: pxc.percona.com/v1
kind: PerconaXtraDBClusterRestore
metadata:
  name: restore1
spec:
  pxcCluster: cluster1
  backupSource:
    destination: operator-testing/cluster1-2022-12-06-19:39:19-full
    azure:
      credentialsSecret: azure-secret
      container: operator-testing
      endpointUrl: https://engk8soperators.blob.core.windows.net
```

Reply:
@egegunes It's wrong. We shouldn't use any prefixes for that